src/cljs/org/broadinstitute/firecloud_ui/common/table.cljs

```diff
@@ -1,18 +1,21 @@
 (ns org.broadinstitute.firecloud-ui.common.table
   (:require
-   [dmohs.react :as react]
-   [org.broadinstitute.firecloud-ui.common :as common]
-   [org.broadinstitute.firecloud-ui.common.components :as comps]
-   [org.broadinstitute.firecloud-ui.common.overlay :as overlay]
-   [org.broadinstitute.firecloud-ui.common.style :as style]
-   [org.broadinstitute.firecloud-ui.common.table-utils :as table-utils]
-   [org.broadinstitute.firecloud-ui.persistence :as persistence]
-   [org.broadinstitute.firecloud-ui.utils :as utils]
-   ))
+    [dmohs.react :as react]
+    [org.broadinstitute.firecloud-ui.common :as common]
+    [org.broadinstitute.firecloud-ui.common.components :as comps]
+    [org.broadinstitute.firecloud-ui.common.overlay :as overlay]
+    [org.broadinstitute.firecloud-ui.common.style :as style]
+    [org.broadinstitute.firecloud-ui.common.table-utils :as table-utils]
+    [org.broadinstitute.firecloud-ui.persistence :as persistence]
+    [org.broadinstitute.firecloud-ui.utils :as utils]
+    ))
 
 
 (def ^:private initial-rows-per-page 10)
 
+(defn select-persistence-keys [atom]
```

Comment:
Should this be private (`defn-`)? I'd also recommend against binding a reserved word (`atom`).

What if you did:
```clojure
(def persistence-keys #{:column-meta :query-params :filter-group-index})
```